specification/machinelearningservices/resource-manager/Microsoft.MachineLearningServices/preview/2023-08-01-preview/machineLearningServices.json

```diff
@@ -862,6 +862,107 @@
       }
     }
   },
+  "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.MachineLearningServices/workspaces/{workspaceName}/computes/{computeName}/allowedVmSizesForResize": {
```

Comment:
>allowedVmSizesForResize [](http://example.com/codeflow?start=164&length=23)

The "allowedVmSizesForResize" is not a valid resource type name. One option is to modify your API path as a POST Action e.g. POST /subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.MachineLearningServices/workspaces/{workspaceName}/computes/{computeName}/getAllowedVmSizesForResize.

Reply:
Could you explain a bit more about why its not an allowed name?